README.md

```diff
@@ -4,7 +4,7 @@ Splunk App to gather insights from your Chef infrastructure via Chef Analytics.
 
 ## Requirements:
 
-* Chef Analytics 1.1.2 or later
+* Chef Analytics 1.1.4 or later
```

Comment:
Yes, exactly, I used that as a placeholder as we likely do not want to release this documentation until we know what release number this is. Right now, our current release is 1.1.3, so, I assume the next release will be 1.1.4